src/services/services.ts

```diff
@@ -1506,7 +1506,21 @@ namespace ts {
 
         function getReferences(fileName: string, position: number, options?: FindAllReferences.Options) {
             synchronizeHostData();
-            return FindAllReferences.findReferencedEntries(program, cancellationToken, program.getSourceFiles(), getValidSourceFile(fileName), position, options);
+
+            // Exclude default library when renaming as commonly user don't want to change that file.

```

Comment:
There's already a feature for that. This will avoid renaming anything on the default library, like comments or strings.